spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -1469,5 +1469,38 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
       }
     }
   }
+  test("unary negative integer overflow test") {
 
+    val df = Seq(Int.MaxValue, Int.MinValue).toDF("a")
+    df.write.mode("overwrite").parquet("/tmp/int.parquet")
```

Comment:
It would be better to use `withTempDir`. There are examples of this in other tests in this suite,